modules/otel-troubleshooting-instrumentation-injection-into-your-workload.adoc

```diff
@@ -0,0 +1,93 @@
+/ Module included in the following assemblies:
+//
+// * observability/otel/otel-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="otel-troubleshooting-instrumentation-injection-into-your-workload_{context}"]
+= Troubleshooting instrumentation injection into your workload
+
+To troubleshoot instrumentation injection, you can perform the following activities:
+
+* Checking if the `Instrumentation` object was created
+* Checking if the init-container started
+* Checking if the resources were deployed in the correct order
+* Searching for errors in the Operator logs
+* Double-checking the pod annotations
+
+.Procedure
+
+. Run the following command to verify that the `Instrumentation` object was successfully created:
++
+[source,console]
+----
+oc get instrumentation -n <workload_project> # <1>
+----
```

Comment:
I'll only leave the comment on this one here, but there are about 7 instances like this where `[source,console]` should be `[source,terminal]`, and the `oc` command is missing its `$` prompt:
```suggestion
[source,terminal]
----
$ oc get instrumentation -n <workload_project> # <1>
----
```
Please apply to all 7 instances.